modules/ocf_backups/manifests/offsite.pp

```diff
@@ -19,5 +28,25 @@
     '/opt/share/backups/boto.cfg':
       source => 'puppet:///private/boto.cfg',
       mode   => '0600';
+
+    # Box.com credentials and API id/secret
+    '/opt/share/backups/box-creds.json':
+      source => 'puppet:///private/box-creds.json',
+      mode   => '0600';
+
+    # This changes each time a backup is made, so it is not stored on the
+    # puppetmaster and instead is just a local file on the backups server.
+    '/opt/share/backups/box-refresh-token':
+      mode => '0600';
```

Comment:
Yeah, pretty much. The issue is that it can't be stored in the puppetmaster's private store from what I can tell because it has to change each time a backup is made, but the token should be kept secure, since if someone has it then they could delete all the backups. I feel like there's a much better way to do this, but I don't know what it would be. Maybe storing the token in the mysql database or on NFS somewhere would be better?